README.adoc

```diff
@@ -1,18 +1,62 @@
+[#trackit-logo]
+image::https://s3-us-west-2.amazonaws.com/trackit-public-artifacts/github-page/logo.png[]
+
 = Trackit (v2)
 
 This is the new iteration of MSolution’s Trackit, a tool to optimize your AWS
 usage and spending.
 
-== Cheking out
+== Don't want to self-host trackit?
+
+We run our own version, check it out at https://trackit.io/[TrackIt.IO]
+
+= Installation
+
+=== #0 Be sure all requirements below are met
+
+- Install https://docs.docker.com/engine/installation/[Docker]
+- Install https://docs.docker.com/compose/install/[Docker Compose]
+- Install https://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS CLI] and https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html[configure your credentials]
+- Install https://golang.org/doc/install[Go]
+- Install https://github.com/kardianos/govendor[govendor]:
+
+[source,sh]
+----
+$> go get -u github.com/kardianos/govendor
+$> export PATH=$PATH:$HOME/go/bin
+----
+
+=== #1 Clone TrackIt
+
+[source,sh]
+----
+$> mkdir -p $HOME/go/src/github.com/trackit && cd $HOME/go/src/github.com/trackit
```

Comment:
This is not portable, you should use `$GOPATH` instead.

Reply:
I used $HOME/go to match how they do it on the [Go Getting started](http://github.com) documentation.
I think users who don't know go will not set the GOPATH since it's not mandatory and they may have issues running the server, whereas people that are familiar with go will understand that they can put it in the GOPATH.
But if you think it's a bad idea I can change it!

Reply:
You need to remember, we need to make it as easy as possible for people to test it. Following the official go documentation to do it sounds like a good principle.

Reply:
Then perhaps the best solution would be to simply provide images for users, and assume people who may want to modify the code know Go and how to manage their `$GOPATH`.